.github/workflows/release-please.yml

```diff
@@ -89,6 +89,7 @@ jobs:
             make update-flagd
             go mod tidy
             make controller-gen
+            CHART_VERSION=${{ needs.release-please.outputs.release_tag_name }} make helm-package
```

Comment:
I dont know if this will work tbh because of the `v` helm is semvar v2

Reply:
See https://github.com/open-feature/open-feature-operator/pull/137#discussion_r1000960319